ingestors/conda_parser.go

```diff
@@ -0,0 +1,63 @@
+package ingestors
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/buger/jsonparser"
+	"github.com/librariesio/depper/data"
+)
+
+var architectures = []string{"linux-32", "linux-64", "linux-aarch64", "linux-armv6l", "linux-armv7l", "linux-ppc64le", "osx-64", "win-64", "win-32", "noarch", "zos-z"}
```

Comment:
Is there a good url with a list of all these arches that we could add in a comment? Might be useful someday if we realize we're missing some and need to add new ones.